shared/src/sources/balancer/graph_api.rs

```diff
@@ -239,7 +344,7 @@ mod tests {
         use pools_query::*;
 
         assert_eq!(
-            serde_json::from_value::<Data>(json!({
+            serde_json::from_value::<Data<WeightedToken>>(json!({
```

Comment:
Can we have a similar test for `Data<StableToken>` with data taken from the subgraph?